### PR TITLE
Motivation, history and Python packaging topic

### PR DESCRIPTION
**Pull request by jimfulton**
This is where, among other things, we say why we use eggs.

### CODE REVIEW

**Comment by jimfulton** [comment]
Thanks for the speedy review.

File: doc/topics/history.rst [hunk 1]
```diff
@@ -0,0 +1,161 @@
...
+Initially, we tried to leverage the ``easy_install`` command
+[#easy_install_module]_, but the goals of ``easy_install`` and
+Buildout were at odds.  ``easy_install`` sought to make it easy for
+humans to install and upgrade packages manually, in an ad hoc manner.
+While it installed dependencies, it didn't upgrade them.  It didn't
+provide ways of managing an installation as a whole.  Buildout, on the
+other hand, was all about automation and repeatability.
+
+To achieve Buildout's goals, it was necessary to interact with
+setuptools at a much lower level and to write quite a bit of packaging
```
Comment (jamadden) [inline doc/topics/history.rst]: quite a bit *more* packaging logic

*more* is missing, *of* no longer needed when added.

File: doc/topics/history.rst [hunk 2]
```diff
@@ -0,0 +1,161 @@
...
+
+Eggs presented a number of challenges, and have a bad reputation as a
+result:
+
+- As an installation format:
+
+  - They needed to be added to the Python path. The ``easy_install``
+    command did this by generating complex `.pth files
+    <https://docs.python.org/2/library/site.html>`_.  This often
+    lead to hard to diagnose bugs and frustration.
```
Comment (jamadden) [inline doc/topics/history.rst]: lead -> led

File: doc/topics/history.rst (hunk 1 above)
Comment (jimfulton) [inline doc/topics/history.rst]: fixed

File: doc/topics/history.rst (hunk 2 above)
Comment (jimfulton) [inline doc/topics/history.rst]: fixed

**Pull request merged**